README.md

```diff
@@ -671,6 +671,9 @@ post.apiDoc = {
 
 ```
 
+Alternatively, args.paths may contain route specifications of the form
```

Comment:
Looks great!  Thank you so much for adding this!  I feel it gets us closer to resolving #12.